src/index.html

```diff
@@ -13,5 +13,182 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="html form">
```

Comment:
Check the code style 
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      name="html form"
    >
```